release/build.sh

```diff
@@ -25,7 +25,7 @@ LOCATION=US
 
 # Set colors if terminal supports it
```

Comment:
nit: Fix shellcheck issues for safety
```
shellcheck build.sh 

In build.sh line 18:
SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
               ^------------------^ SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: 
SCRIPT_DIR="$( cd "$(dirname "$0")" || exit ; pwd -P )"


In build.sh line 81:
  if ! bq show --headless $dataset > /dev/null 2>&1; then
                          ^------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  if ! bq show --headless "$dataset" > /dev/null 2>&1; then


In build.sh line 82:
    bq mk --headless -d --data_location=${LOCATION} ${dataset}
                                                    ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    bq mk --headless -d --data_location=${LOCATION} "${dataset}"


In build.sh line 105:
  printf "${BOLD}${file}${NORMAL}\n"
         ^-------------------------^ SC2059: Don't use variables in the printf format string. Use printf '..%s..' "$foo".


In build.sh line 107:
    bq query --dataset_id ${dataset} --headless --nouse_legacy_sql --dry_run "$(cat ${file})"
                          ^--------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                    ^-----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    bq query --dataset_id "${dataset}" --headless --nouse_legacy_sql --dry_run "$(cat "${file}")"


In build.sh line 109:
    bq query --dataset_id ${dataset} --headless --nouse_legacy_sql "$(cat ${file})"
                          ^--------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                          ^-----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    bq query --dataset_id "${dataset}" --headless --nouse_legacy_sql "$(cat "${file}")"


In build.sh line 112:
  if [[ $? -gt 0 ]]; then
        ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.


In build.sh line 113:
    printf "Failed to create: $file"
           ^-----------------------^ SC2059: Don't use variables in the printf format string. Use printf '..%s..' "$foo".


In build.sh line 131:
  local sql_files=$(find ${UDF_DIR} -type f -name "*.sql")
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 132:
  local num_files=$(echo "$sql_files" | wc -l)
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 134:
    sed -i "s|\${JS_BUCKET}|${JS_BUCKET}|g" ${file}
                                            ^-----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    sed -i "s|\${JS_BUCKET}|${JS_BUCKET}|g" "${file}"


In build.sh line 142:
  --substitutions _JS_BUCKET=${JS_BUCKET},SHORT_SHA=${SHORT_SHA}
                             ^----------^ SC2086: Double quote to prevent globbing and word splitting.
                                                    ^----------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  --substitutions _JS_BUCKET="${JS_BUCKET}",SHORT_SHA="${SHORT_SHA}"


In build.sh line 145:
  if [[ $? -gt 0 ]]; then
        ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.


In build.sh line 173:
  local sql_files=$(find . -type f -name "*.sql")
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 174:
  local num_files=$(echo "$sql_files" | wc -l)
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 176:
  printf "Building $num_files database objects...\n"
         ^-- SC2059: Don't use variables in the printf format string. Use printf '..%s..' "$foo".


In build.sh line 178:
    local dataset=$(get_dataset ${file})
          ^-----^ SC2155: Declare and assign separately to avoid masking return values.
                                ^-----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    local dataset=$(get_dataset "${file}")


In build.sh line 180:
    if [[ ! -z ${dataset} ]]; then
          ^-- SC2236: Use -n instead of ! -z.


In build.sh line 181:
      create_dataset_if_not_exists ${dataset}
                                   ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
      create_dataset_if_not_exists "${dataset}"


In build.sh line 182:
      execute_query ${file} true ${dataset}
                    ^-----^ SC2086: Double quote to prevent globbing and word splitting.
                                 ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
      execute_query "${file}" true "${dataset}"


In build.sh line 200:
  local files_changed=$(git diff --name-only origin/master)
        ^-----------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 204:
  if [[ ! -z $(echo ${files_changed} | grep udfs/) ]]; then
        ^-- SC2236: Use -n instead of ! -z.
          ^-- SC2143: Use ! grep -q instead of comparing output with [ -z .. ].
                    ^--------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  if [[ ! -z $(echo "${files_changed}" | grep udfs/) ]]; then


In build.sh line 223:
  local sql_files=$(find ${UDF_DIR} -type f -name "*.sql")
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 224:
  local num_files=$(echo "$sql_files" | wc -l)
        ^-------^ SC2155: Declare and assign separately to avoid masking return values.


In build.sh line 226:
  printf "Creating or updating $num_files database objects...\n"
         ^-- SC2059: Don't use variables in the printf format string. Use printf '..%s..' "$foo".


In build.sh line 228:
    local dataset=$(get_dataset ${file})
          ^-----^ SC2155: Declare and assign separately to avoid masking return values.
                                ^-----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
    local dataset=$(get_dataset "${file}")


In build.sh line 231:
      create_dataset_if_not_exists ${dataset}
                                   ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
      create_dataset_if_not_exists "${dataset}"


In build.sh line 232:
      execute_query ${file} false ${dataset}
                    ^-----^ SC2086: Double quote to prevent globbing and word splitting.
                                  ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
      execute_query "${file}" false "${dataset}"


In build.sh line 250:
  cd ${SCRIPT_DIR}/.. || exit
     ^-----------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  cd "${SCRIPT_DIR}"/.. || exit

For more information:
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
  https://www.shellcheck.net/wiki/SC2164 -- Use 'cd ... || exit' or 'cd ... |...
  https://www.shellcheck.net/wiki/SC2059 -- Don't use variables in the printf...
```

Reply:
Done.